miden-lib/src/tests/test_tx.rs

```diff
@@ -4,16 +4,21 @@ use miden_objects::{
 };
 use mock::{
     constants::ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN,
-    mock::{account::MockAccountType, notes::AssetPreservationStatus, transaction::mock_inputs},
+    mock::{
+        account::MockAccountType, host::MockHost, notes::AssetPreservationStatus,
+        transaction::mock_inputs,
+    },
     prepare_transaction,
     procedures::prepare_word,
     run_tx, run_within_tx_kernel,
 };
 
-use super::{ContextId, Felt, MemAdviceProvider, ProcessState, StackInputs, Word, ONE, ZERO};
+use super::{
+    ContextId, Felt, MemAdviceProvider, Process, ProcessState, StackInputs, Word, ONE, ZERO,
+};
 use crate::transaction::memory::{
-    CREATED_NOTE_ASSETS_OFFSET, CREATED_NOTE_METADATA_OFFSET, CREATED_NOTE_RECIPIENT_OFFSET,
-    CREATED_NOTE_SECTION_OFFSET, NUM_CREATED_NOTES_PTR,
+    CREATED_NOTE_ASSETS_OFFSET, CREATED_NOTE_METADATA_OFFSET, CREATED_NOTE_NUM_ASSETS_OFFSET,
```

Comment:
Also here, we have a lot of `CREATED_NOTE` instead of `OUTPUT_NOTE`, I think it would be easier to follow if we change it everywhere.

Reply:
Yep - leaving off comprehensive renaming to the future PR as it will touch a lot of things.